packages/plugin-core/src/_extension.ts

```diff
@@ -972,6 +973,16 @@ async function _setupCommands(
 
   ALL_COMMANDS.map((Cmd) => {
     const cmd = new Cmd();
+
+    // Register commands that implement on `onAutoComplete` with AutoCompletableRegister
```

Comment:
Rather than needing a ts-ignore, we should be able to create a user defined type guard fn for the AutoCompletable type/interface. https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards